images/nested-podman-entrypoint.sh

```
@@ -0,0 +1,52 @@
+# Create User ID entry if running as an unmapped UID (common in OpenShift)
+if ! whoami &> /dev/null; then
+  if [ -w /etc/passwd ]; then
+    echo "${USER_NAME:-user}:x:$(id -u):0:${USER_NAME:-user} user:${HOME}:/bin/bash" >> /etc/passwd
+    echo "${USER_NAME:-user}:x:$(id -u):" >> /etc/group
+  fi
+fi
+
+# Create subuid/gid entries for rootless podman
+USER=$(whoami)
+START_ID=$(( $(id -u)+1 ))
+END_ID=$(( 65536-${START_ID} ))
```

Shellcheck (0.11.0)

[style] $/${} is unnecessary on arithmetic variables.

(SC2004)